i18n/es/translation.json

```
@@ -551,5 +551,6 @@
   "Scheduled Transactions": "Transacciones Programadas",
   "Executed Transactions": "Transacciones Ejecutadas",
   "You can withdraw all streams at once.": "Puedes retirar todos los streams a la vez.",
-  "Whitdraw All": "Retirar Todo"
+  "Whitdraw All": "Retirar Todo",
```

I know it's not change from this PR, but there's a typo here
```suggestion
  "Withdraw All": "Retirar Todo",
```

---

thanks, i also updated the typo in the component